validator/client/src/test/java/tech/pegasys/teku/validator/client/ValidatorIndexProviderTest.java

```diff
@@ -61,7 +61,8 @@ void shouldLoadValidatorKeys() {
     when(validatorApiChannel.getValidatorIndices(validators.getPublicKeys()))
         .thenReturn(SafeFuture.completedFuture(Map.of(key1, 1, key2, 20, key3, 300)));
     provider.lookupValidators();
-    assertThat(provider.getValidatorIndices()).isCompletedWithValue(IntArrayList.of(1, 20, 300));
+    final IntCollection actual = provider.getValidatorIndices().getImmediately();
+    assertThat(actual).containsExactlyInAnyOrderElementsOf(IntArrayList.of(1, 20, 300));
```

Comment:
why this change? (the same below)

Reply:
it's not related to this PR but this test is not passing on my computer without this change
I could do it in separate PR

Reply:
I will move it to separate PR

Reply:
Merged in #9010